CIPs/cip-0024.md

```diff
@@ -21,9 +20,9 @@ Enable the activation of the Celo Churrito Hardfork specification which includes
 Add support for protocol-impacting changes introduced in Geth in a hardfork called `Churrito`.
 
 This document proposes the following blocks at which to implement these changes in the Celo networks:
-- `2598139` on Baklava Testnet (Tuesday November 24th, 2020)
-- `2320192` on Alfajores Testnet (Tuesday December 1st, 2020)
-- `3976252` on Celo Mainnet (Tuesday December 8th, 2020)
+- `2840059` on Baklava Testnet (Tuesday December 8th, 2020)
```

Comment:
Let's move Baklava to Dec 1st instead (block number 2719099 according to my calculation).  That way we increase the space between Baklava and Mainnet from 2 weeks to 3 weeks.